src/xdocs-test/resources/com/puppycrawl/tools/checkstyle/checks/naming/abstractclassname/XDocExample1.java

```diff
@@ -0,0 +1,17 @@
+/*
+AbstractClassName
+format = (default)^Abstract.+$
+ignoreModifier = (default)false
+ignoreName = (default)false
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.naming.abstractclassname;
+
+class XDocExample1 {
+    abstract class AbstractFirstClass {} // OK
+    abstract class SecondClass {} // violation '.* must match pattern '\^Abstract\.\+\$''
```

Comment:
we will merge this Check as first and real representation of how it will looks like.

Next 2 PRs should be selecteted for Check that have midsize example and massive example. To let us simulate in real how it will looks like after migration of all.

Reply:
> We should remove "package" in HTML version of this snippet.

This is a job for the input parser, correct?

Reply:
> Please do indentation as +2.

Done.

Reply:
> lets manually move this example to javadoc and xdoc to see how it look like in final rendering - html.

Done. Please check at https://stoyank7.github.io/config_naming.html#AbstractClassName_Examples

![Screenshot from 2023-05-31 19-57-44](https://github.com/checkstyle/checkstyle/assets/23459549/3ff120e0-6445-4b42-9f1a-698148dcbb76)

Reply:
>This is a job for the input parser, correct?

Yes, just create issue for it to not forget.
we do 3 manual to catch what we need to do.

------

Please remove `//Ok` comments, they are proven by time to be just noise.

Reply:
@stoyanK7 , please do this

Reply:
@romani Done.